src/components/GroveNavbar.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Navbar as BSNavbar } from 'react-bootstrap';
+import { UserInfo } from './UserInfo';
+
+export const GroveNavbar = ({
```

Comment:
Rename this to Navbar to make the hierarchy less confusing.  This will be the bootstrap component wrapper.